### PR TITLE
Fix social icons

### PR DESCRIPTION
**Pull request by ghost**
### Screenshoots
Before:
image

After:
image

### Changes in the PR
- Updated CSS and HTML of social icons for mobile devices
- Extracted javascript code in js files
- Deleted extra title called `Tutorials` in Tutorials main page (bug when posts aren't filtered)
- Deleted extra title in [Connecting a Haskell Backend to a PureScript Frontend](https://www.stackbuilders.com/tutorials/functional-full-stack/purescript-bridge/)

### CODE REVIEW

**Review by CristhianMotoche — APPROVED** [review]
As we discussed this is for the title of the blog posts filtered. The title that looks like this:
```
Posts tagged "haskell"
```

Well, if we plan to remove that no problem. I'll approve this.

File: app/Main.hs [hunk 1]
```diff
@@ -92,7 +95,7 @@ rules env = do
         indexContext =
           listField "tutorials" tutorialCtx (return tutorials)
             <> constField "title" "Home"
-            <> constField "title-list" "Tutorials"
+            <> constField "title-list" ""
```
Comment (CristhianMotoche) [inline app/Main.hs:98]: I think an extra space will be added anyway, right?

**Review by CristhianMotoche — APPROVED** [review]
LGTM! Good job :shipit:

File: app/Main.hs (hunk 1 above)
Comment (ghost) [reply app/Main.hs:98]: Yes, but the best way is add a conditional in template, because if is a space, it will create a `h1` empty tag. I will fix it.

Comment (ghost) [reply app/Main.hs:98]: Forget it, Hakyll doesn't allow comparisons in conditionals: https://jaspervdj.be/hakyll/tutorials/04-compilers.html#conditionals

**Pull request merged**